src/components/reference-book/reference-book.cjsx

```diff
@@ -5,27 +5,42 @@ _  = require 'underscore'
 
 BindStoreMixin = require '../bind-store-mixin'
 NavBar = require './navbar'
+Menu = require './slide-out-menu'
 
 {CourseListingStore} = require '../../flux/course-listing'
 {CourseStore} = require '../../flux/course'
 
 module.exports = React.createClass
   displayName: 'ReferenceBook'
+  contextTypes:
+    router: React.PropTypes.func
 
-  mixins: [Router.State, BindStoreMixin]
+  mixins: [BindStoreMixin]
   bindStore: CourseListingStore
   bindEvent: 'loaded'
 
   componentWillMount: ->
     CourseListingStore.ensureLoaded()
 
+  getInitialState: ->
+    {cnxId} = @context.router.getCurrentParams()
+    # Pop open the menu unless the page was explicitly navigated to
+    {isMenuVisible: not cnxId}
+
   showTeacherEdition: ->
     @setState(showTeacherEdition: not @state.showTeacherEdition)
 
+  onMenuClick: ->
+    @toggleMenuState() unless window.innerWidth > 1350
```

Comment:
Can you make this a const on top of the file and add a searchable string to find the corresponding values in the LESS file?